backend/app/celery_worker.py

```diff
@@ -0,0 +1,13 @@
+# Use for local development only
+import os
+from dotenv import load_dotenv
+
+
+if __name__ == '__main__':
+    # Load environment variables from .env file
+    dotenv_path = os.path.join(os.path.dirname(__file__), '../../docker/.env-local')
+    load_dotenv(dotenv_path)
+
+    # Start the Celery worker
+    from app.worker.app import celery_app
+    celery_app.worker_main(['--app', 'app.worker.app.celery_app', 'worker', '-l', 'info', '-c', '4', '-Ofair', '--without-heartbeat', '--without-gossip', '--without-mingle'])
```

Comment:
The reason for not using the default value is that the Swiple jobs are mostly I/O and not CPU-bound. For this reason, I think it would be beneficial to have some level of concurrency. Concurrency is only important when Swiple is deployed to a production environment. Currently, only 1 CPU is needed to run a Swiple worker but we want to be able to run jobs concurrently.

These options were applied after reading the following Medium post https://medium.com/pythonistas/a-complete-guide-to-production-ready-celery-configuration-5777780b3166. I'm relying on his knowledge and trying to provide a "production ready" implementation. Perhaps, they should only be disabled if we run into performance issues?

Reply:
Hmm... Then, if your job is mostly I/O, you may benefit from using another type of processing pool. By default, it's `prefork`, which is a classic multiprocess approach. For I/O intensive tasks, it may be interesting to switch to `eventlet` or `gevent`. You can read more details about it here: https://distributedpython.com/posts/celery-execution-pools-what-is-it-all-about/

Reply:
Yes agreed! Originally, I tried using `eventlet` and `gevent` but something was blocking the main thread in `great-expectations` so I had to go with the classic multiprocess approach unfortunately.